package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@pgateway/common-services-api",
   "main": "dist/index.js",
   "type": "dist/index.d.ts",
-  "version": "2.4.0",
-  "description": "Utlity to connect to AWS API GW",
+  "version": "2.5.0",
```

Comment:
Just a suggestion: I think that version bumping is a big action, so we might want to consider separating it out. This might also be why we left out the publishing of v2.4.0 previously. We can work out a good procedure eventually. In the meantime, keeping it like this PR is okay as long as we deploy it correctly (which we will since we need to use it now haha). Otherwise, we can do like [pg-eslint-config](https://github.com/pway123/pg-eslint-config/commits/master) and use a separate commit for it.